From 20d105b96df404cde10c4fcaece6331be88ba25a Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@kernel.org>
Date: Tue, 31 Aug 2021 14:56:57 -0500
Subject: [PATCH 10/20] EDAC/synopsys: v3.80a of the synopsys edac contoller is
 also on the N5X

[upstream commit 8d6becc28c6a0d57267bf6f1beaa4659e294683b]

commit cba959948dce285cb7625e047ba585bd385f7bc8 from
https://github.com/altera-opensource/linux-socfpga/commits/socfpga-5.4.124-lts

Intel's N5X platform is also using the Synopsys EDAC controller.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/edac/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/edac/Kconfig b/drivers/edac/Kconfig
index bbac8a3e4518..11c47f12cfc0 100644
--- a/drivers/edac/Kconfig
+++ b/drivers/edac/Kconfig
@@ -471,7 +471,7 @@ config EDAC_ALTERA_SDMMC
 
 config EDAC_SYNOPSYS
 	tristate "Synopsys DDR Memory Controller"
-	depends on ARCH_ZYNQ || ARCH_ZYNQMP
+	depends on ARCH_ZYNQ || ARCH_ZYNQMP || ARCH_N5X
 	help
 	  Support for error detection and correction on the Synopsys DDR
 	  memory controller.
-- 
2.31.1

